### ISSUE TITLE
command bugs

### ISSUE THREAD

**Issue by cornernote** [cornernote/minetest-skyblock#40]
`/who` - doesnt fit into the chat, only shows like 5-6 lines

`/level <player>` - doesnt show the player, shows your own level

**Comment by LeMagnesium** [comment]
About `/level <player>`, I fixed that.  Regarding `/who`, as you can and must do after typing `/help all`, you can just use F10 to see the output in the console.

**Comment by cornernote** [comment]
Awesome.  I didn't know about F10.

I think your fix for level broke the ability to change the players level, eg `/level <player> [level]` (`[level]` is optional).

**Comment by cornernote** [comment]
Perhaps it should be a different command to set the players level... perhaps `/setlevel`, then its less effort to mess with the regex.

**Comment by LeMagnesium** [comment]
Yes, I didn't know level was intended to change a player's level. I'll change that in my PR then. `/level` would give someone's level, without any privilege needed, and `/setlevel` would set someone's level, requiring `level`privilege.
